packages/server/src/routes/index.js

```diff
@@ -14,9 +14,6 @@ router.use(authorizer, (req, res, next) => {
       clientId: process.env.CLIENT_ID,
       uri: process.env.REDIRECT_URI
     },
-    {
-      apiHostname: process.env.OAUTH_URL
```

Comment:
OAUTH_URL will only be used for `logout`?

Reply:
Not attempted to follow all the code ... but our mobile apps target `login.` for the web login flow and for access token management after that point in time. We only target `api.` with actual API requests, that was the ask a while back. Use login for auth stuff.

Reply:
@shawnce thats what we are doing with this sample app as well. The server portion of the app takes two configuration values: `BASE_URL` and `OAUTH_URL`. The base value is used for all API requests, while the oauth value is used for the initial login and token management. 

The sdk client options (https://github.com/procore/js-sdk/blob/master/lib/clientOptions.ts) takes in a configuration value for the base of the URL that is used for every request. Ex authorize: https://github.com/procore/js-sdk/blob/master/lib/authorize.ts#L9